libraries/standard/http/cbmc/proofs/demo/Makefile

```diff
@@ -0,0 +1,82 @@
+METHOD=IotMqtt_Disconnect
+ENTRY=harness
+
+################################################################
+# Proof assumptions
+
+# Bound the number of subscriptions in a list (BOUND = MAX-1)
+SUBSCRIPTION_COUNT_MAX=2
+DEF += -DSUBSCRIPTION_COUNT_MAX=$(SUBSCRIPTION_COUNT_MAX)
```

Comment:
I wonder if there is a nice macro to do this?

Reply:
Do you mean macro that would take SUB_CNT and 2 and produce the 2 lines?  I'm inclined to leave this as it is, I think macros can obfuscate, and I prefer to leave the macros for things that must be right (eg, how to invoke cbmc) and not for stuff like this that most programmers see every day.